layer/projects/progress_tracker_updater.py

```diff
@@ -159,6 +159,9 @@ def _handle_task_succeeded(self, task: PBTask) -> None:
                 train_index=str(train_index),
                 version=version_name,
             )
+            self.tracker.mark_model_saved(
```

Comment:
I think we should only send one event to the tracker here, either `trained` or `saved` but not both.